build/generateIcons.js

```diff
@@ -0,0 +1,62 @@
+
+try {
+    console.log(' ');
+    console.log(' -------------------------------------------------------------------------------');
+    console.log('  ICONS GENERATOR - PROCESS START');
+    console.log(' -------------------------------------------------------------------------------');
+    console.log(' ');
+    const webfontsGenerator = require('vusion-webfonts-generator');
```

Comment:
can you use es6 syntax  for imports?

Reply:
I'm using require because some versions of node doesn't not support `import`